generators/server/templates/src/test/java/package/service/UserServiceIntTest.java.ejs

```diff
@@ -400,12 +400,6 @@ public class UserServiceIntTest <% if (databaseType === 'cassandra') { %>extends
     @Transactional
     <%_ } _%>
     public void assertThatUserLocaleIsCorrectlySetFromAuthenticationDetails() {
-        user.setId(Constants.ANONYMOUS_USER);
-        user.setLogin(Constants.ANONYMOUS_USER);
```

Comment:
Leaving lines 403 and 404 seems to work based on a test I did ([Travis Results](https://travis-ci.org/ruddell/generator-jhipster/builds/431242491), [Commit](https://github.com/ruddell/generator-jhipster/commit/f3ae6d8a7ef9))